libr/anal/p/anal_arm_cs.c

```diff
@@ -890,6 +890,10 @@ static int analop64_esil(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int l
 		r_strbuf_setf (&op->esil,
 			"%s,%s,*,%s,+,%s,=",REG64(2),REG64(1),REG64(3), REG64(0));
 		break;
+	case ARM64_INS_MSUB:
+		r_strbuf_setf (&op->esil,
+			"%s,%s,*,%s,-,%s,=",REG64(2),REG64(1),REG64(3), REG64(0));
```

Comment:
add spaces after commas, maybe the format string fits in the line above and will be more readable